manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,23 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    if $::osfamily == 'RedHat' {
+      $rh_ruby_prefix = 'rubygem'
+      if $::operatingsystem != 'Fedora' {
+        $ruby_prefix = "tfm-${rh_ruby_prefix}"
+      } else {
+        $ruby_prefix = $rh_ruby_prefix
+      }
+    } else {
+      $ruby_prefix = 'ruby'
+    }
+
+    package { "${ruby_prefix}-net-ssh-krb":
```

Comment:
Should this restart the dynflow_core service? Note that it's only used on EL and not Fedora or other distro's.